collect_app/src/main/java/org/odk/collect/android/widgets/ExStringWidget.java

```diff
@@ -284,6 +289,7 @@ private void onException(String toastText) {
                 toastText, Toast.LENGTH_SHORT)
                 .show();
         this.answer.requestFocus();
```

Comment:
You show keyboard for answer EditText so I think you can move this line to the showKeyboard() method, and you can get rid of `this.`